src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  ForwardRefRenderFunction,
+  PropsWithChildren,
+  forwardRef,
+  useState,
+} from 'react';
+import { transparentize } from 'polished';
+import styled from '@emotion/styled';
+
+const SMALL = 'small';
+const MEDIUM = 'medium';
+const LARGE = 'large';
+
+const defaultAvatarSizes = {
+  small: 32,
+  medium: 115,
+  large: 150,
+};
+
+type sizeType = typeof SMALL | typeof MEDIUM | typeof LARGE;
+type AvatarProps = {
+  alt: string;
+  src: string;
+  size: sizeType;
+};
```

Comment:
I would rewrite it like below:
```suggestion
const AvatarSize = {
  SMALL: 'small',
  MEDIUM: 'medium',
  LARGE: 'large'
} as const;

type SizeTypeProps = keyof typeof AvatarSize;

type AvatarSizeValueProps = typeof AvatarSize[SizeTypeProps]

type DefaultAvatarSize = {
  [KDefaultSize in AvatarSizeValueProps]: number;
};

const defaultAvatarSizes: DefaultAvatarSize = {
  small: 32,
  medium: 115,
  large: 150,
};

type AvatarProps = {
  alt: string;
  src: string;
  size: AvatarSizeValueProps;
};
```